content/tutorials/quickstart/batch-minting.md

```diff
@@ -0,0 +1,494 @@
+
+## Get Account From Seed
+
+This is a stripped-down version of `getAccountsFromSeeds()` that only gets the Standby account information.
+
+```javascript
+async function getAccountFromSeed() {
+```
+
+ Connect to the XRP Ledger.
+
+```javascript
+	let net = getNet()
+	const client = new xrpl.Client(net)
+	results = 'Connecting to ' + getNet() + '....'
+	document.getElementById('standbyResultField').value = results
+	await client.connect()
+	results += '\nConnected, finding wallets.\n'
+	document.getElementById('standbyResultField').value = results
+```
+
+Use the account seed to get information about the wallet.
+
+```javascript
+  var theSeed = document.getElementById('seeds').value
+	const standby_wallet = xrpl.Wallet.fromSeed(theSeed)
+```
+
+Get the current XRP balance.
+
+```javascript	
+  const standby_balance = (await client.getXrpBalance(standby_wallet.address))  
+```	
+
+Populate the fields for Standby account.
+
+```javascript
+	document.getElementById('standbyAccountField').value = standby_wallet.address
+	document.getElementById('standbyPubKeyField').value = standby_wallet.publicKey
+	document.getElementById('standbyPrivKeyField').value = standby_wallet.privateKey
+	document.getElementById('standbySeedField').value = standby_wallet.seed
+	document.getElementById('standbyBalanceField').value = 
+		(await client.getXrpBalance(standby_wallet.address))
+```
+
+Disconnect from the XRP Ledger.
+
+```javascript
+ client.disconnect()	
+} 
+```
+
+## Get Batch NFTokens
+
+This version of `getTokens()` allows for a larger set of NFTokens by watching for a `marker` at the end of each batch of NFTokens.
+
+```javascript
+async function getBatchNFTokens() {
+```
+
+Connect to the XRPLedger and get the account wallet.
+
+```javascript
+  const standby_wallet = xrpl.Wallet.fromSeed(standbySeedField.value)
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  results = 'Connecting to ' + net + '...'
+  document.getElementById('standbyResultField').value = results
+  await client.connect()
+  results += '\nConnected. Getting NFTokens...'
+  document.getElementById('standbyResultField').value = results
+```
+
+Request the `account_nfts`. Set the `limit` to 400, the maximum amount, to retrieve up to 400 records in one batch.
+
+```javascript
+  results += "\n\nNFTs:\n"
+  let nfts = await client.request({
+    method: "account_nfts",
+    account: standby_wallet.classicAddress,
+    limit: 400
+  })
+  results += JSON.stringify(nfts,null,2)
+```
+
+If the list of NFTokens exceeds your limit, the result will include a `marker` field that you can use as a parameter for the next `account_nfts` request. The `marker` indicates where the next batch of records starts. While the `marker` field is not null, continue to request another batch of NFToken records.
+
+```javascript
+  while (nfts.result.marker != null)
```

Comment:
```suggestion
  while (nfts.result.marker)
```

The expression as written misleadingly implies that the API returns the actual JSON value `null` in the marker field when there are no additional pages. I don't think this is the case. If it's like any of the other methods, the API omits the `nfts.results.marker` field entirely when there are no additional pages. I suspect that the code as written only works (if it even does!) because JavaScript implicitly converts the value `undefined` to something that it considers equivalent to `null` in this operation.

But I don't have an account with 401+ NFTs on hand to test with, so I'm not sure.